server/routes/deprecated/dashboardWithoutPaginationView.ts

```diff
@@ -48,8 +48,8 @@ export default class DashboardWithoutPaginationView {
       },
 
       autocomplete: 'off',
-      id: 'open-case-search-text',
-      name: 'open-case-search-text',
+      id: 'case-search-text',
+      name: 'case-search-text',
```

Comment:
This is in a `/deprecated/` path -- do we need this file?

Reply:
@sldblog Looks from a glance at the imports that this file is still used in serviceProviderReferralsDashboard - hence needs to be modified to keep the changes working.